packages/medusa/src/loaders/plugins.js

```diff
@@ -16,6 +16,7 @@ import fs from "fs"
 import { asValue, asClass, asFunction, aliasTo } from "awilix"
 import { sync as existsSync } from "fs-exists-cached"
 
+import { AbstractTaxService } from "../interfaces/tax-service"
```

Comment:
I notice all the rest of these AbstractServices are in `medusa-interfaces`. Doesn't matter to me where it lives as long as it's exported, just wanted to point that out.